app/decorators/order_detail_decorator.rb

```diff
@@ -0,0 +1,6 @@
+class OrderDetailDecorator < SimpleDelegator
+
+  delegate :full_name, to: :assigned_user, prefix: :assigned_user, allow_nil: true
+  delegate :name, to: :order_status, prefix: :order_status, allow_nil: true
```

Comment:
Yeah, I guess we could use `to_s`.  I noticed `to_s` and thought it would be less clear of what was going on in the view.  But now that I think about it some more, I guess it's not really a code small to use the overridden `to_s` and forego the decorator.  Do you think we'll need this decorator for OrderDetail in the future, I assumed yes since the model is so fat.  But haven't taken the time to verify.